src/raftstore/store/peer.rs

```diff
@@ -1052,6 +1055,18 @@ impl Peer {
         None
     }
 
+    pub fn heartbeat_pd(&self, cfg: &Config, worker: &Worker<PdTask>) {
```

Comment:
I see you still use heartbeat in store, so why removing this function in store and add it here?